lib/api/bucketGet.js

```diff
@@ -63,9 +238,13 @@ export default function bucketGet(authInfo, request, log, callback) {
         delimiter: params.delimiter,
         marker: params.marker,
         prefix: params.prefix,
+
     };
+    if (versions) {
+        listParams.versions = versions;
+    }
```

Comment:
Actually I do use `versions` in `listParams` to inform metadata that this is a listing of all versions.